src/types/commands.ts

```diff
@@ -495,8 +495,21 @@ export interface UpdateTableCommand {
   config?: Partial<TableConfig>;
 }
 
+export interface ResizeTableCommand {
+  type: "RESIZE_TABLE";
+  zone: Zone;
+  sheetId: UID;
+  newTableRange: RangeData;
+  tableType?: CoreTableType;
```

Comment:
It's needed because `tableType` is needed for the UPDATE_TABLE command. It's the responsability of the command to tell wether the table is dynamic or not, because the core plugin cannot know wether the table can be dynamic or not.